docs/download_files.md

```diff
@@ -95,32 +95,51 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
 
 ## Portal-wide downloads
 
-<!--TODO: Fill in information about portal-wide downloads and provide screenshots of all file structure downloads -->
+The Portal-wide download page can be used to instantly download all [metadata](#metadata-only-downloads) or gene expression data for all samples on the Portal. 
 
-### Metadata-only downloads
-
-The Portal-wide metadata download is a single TSV file containing the metadata for all samples with associated single-cell RNA-seq, single-nuclei RNA-seq, or spatial transcriptomics data available on the Portal.
-A table describing all columns included in the file can be found in the [metadata section below](#metadata).
+All single-cell and single-nuclei gene expression data from the Portal can be downloaded as a single zip file containing data stored as either [`SingleCellExperiment` objects (`.rds` files)](#singlecellexperiment-portal-wide-download-structure) or [`AnnData` objects (`.h5ad` files)](#anndata-portal-wide-download-structure). 
+All spatial data for any samples sequenced using [spatial transcriptomics](#spatial-transcriptomics-libraries) is available separately as a zip file. 
 
-### `SingleCellExperiment` downloads
+When downloading any of the available Portal-wide data downloads all metadata and associated bulk RNA-seq data is also included. 
```

Comment:
I'm not sure "associated" makes sense here, since it's everything? 
```suggestion
When downloading any of the available Portal-wide data downloads, all metadata and bulk RNA-seq data is also included. 
```

Reply:
I think I used "associated", since I think for the spatial data it's only bulk for samples with spatial data, but maybe you're right and it is everything. Either way, I think this edit cleans up the sentence and is probably less confusing.